docs/getting-started/golden-path.md

```diff
@@ -23,10 +23,12 @@ Continue to download and install...
 
 ***MAC***
 
-* [Decrediton](https://github.com/decred/decrediton)
+* [Download](https://github.com/decred/decred-binaries/releases) and [install](https://github.com/decred/decrediton) Decrediton then
```

Comment:
I think for install you should link directly to the install section of the readme instead of to the repo
https://github.com/decred/decrediton/blob/master/README.md#installation